server/protocols/jmap-integration-testing/cassandra-jmap-integration-testing/src/test/java/org/apache/james/jmap/cassandra/cucumber/CassandraStepdefs.java

```diff
@@ -70,9 +78,15 @@ public void init() throws Exception {
             .build();
 
         mainStepdefs.jmapServer = new GuiceJamesServer(configuration)
-                .combineWith(ALL_BUT_JMX_CASSANDRA_MODULE)
-                .overrideWith(new CassandraJmapServerModule(embeddedElasticSearch, cassandraServer.getHost()))
-                .overrideWith((binder) -> binder.bind(PersistenceAdapter.class).to(MemoryPersistenceAdapter.class));
+            .combineWith(ALL_BUT_JMX_CASSANDRA_MODULE)
```

Comment:
How are these changes related to the PR ?

Reply:
`CassandraJmapServerModule` was used only here and **messed things up**.

I remove it as it bringed no value. And rather write the needed overrides here.